docs/src/app_dev/daml_api/token_standard.rst

```diff
@@ -0,0 +1,291 @@
+* `To reject a TransferInstruction </app_dev/token_standard_openapi/index.html#post--registry-transfer-instruction-v1-transferInstructionId-choice-contexts-reject>`_
+* `To withdraw a TransferInstruction </app_dev/token_standard_openapi/index.html#post--registry-transfer-instruction-v1-transferInstructionId-choice-contexts-withdraw>`_
+* `To withdraw an Allocation </app_dev/token_standard_openapi/index.html#post--registry-allocations-v1-allocationId-choice-contexts-withdraw>`_
+* `To cancel an Allocation </app_dev/token_standard_openapi/index.html#post--registry-allocations-v1-allocationId-choice-contexts-cancel>`_
+
+The response of these endpoints include two fields:
+
+* ``choiceContextData``: to be passed as ``context`` in the ``choiceArgument``.
+* ``disclosedContracts``: to be passed in the submit or prepare request.
+
+.. warning::
+
+  Note that ``AllocationRequest_Reject`` and ``AllocationRequest_Withdraw`` should be called with an empty choice context.
+  This is currently there as a potential future extension.
+
+
+
+.. _token_standard_usage_workflows:
+
+Token Standard Workflows
+------------------------
+
+Transfers
+^^^^^^^^^
+
+The workflow will follow the following steps:
+
+* A party calls ``TransferFactory_Transfer`` to create a transfer.
+* Depending on the implementation of the factory, one of three ``TransferInstructionResult_Output`` is possible:
+
+  * ``Failed``: where the transfer did not succeed and all holdings (minus fees) have been returned to the sender.
+  * ``Completed``: where the transfer succeeded and the receiver has received their holdings. No further action is required.
+  * ``Pending``: where the transfer is pending further steps. This will include a ``transferInstructionCid``.
+* If ``Pending``, The receiver party observes a ``TransferInstruction`` (which has the same contract id as above). Then:
+
+  * The receiver can exercise ``TransferInstruction_Accept``, which again will return a ``TransferInstructionResult`` depending on success and whether further steps are required or not.
+  * The receiver can exercise ``TransferInstruction_Reject``, same as above.
+  * The sender can exercise ``TransferInstruction_Withdraw``, again returning a ``TransferInstructionResult``.
+  * The registry can exercise ``TransferInstruction_Update``, again returning a ``TransferInstructionResult``.
+
+Allocations
+^^^^^^^^^^^
+
+The workflow will follow the following steps:
+
+* A registry creates as many ``AllocationInstructions`` as required for a workflow to happen.
+* Parties can:
+
+  * The registry can exercise ``AllocationInstruction_Withdraw`` or ``AllocationInstruction_Update`` on it.
+  * The senders of each transfer leg can exercise ``AllocationFactory_Allocate`` to create an ``Allocation`` satisfying the conditions of the ``AllocationInstruction``.
+  * The following choices can be called on the ``Allocation``:
```

Comment:
Consider referencing the source code and its commentary. So that people become accustomed to actually reading it and using it as guidance.